crates/genesis/src/chain.rs

```diff
@@ -55,6 +55,19 @@ pub struct HardForkConfiguration {
     pub holocene_time: Option<u64>,
 }
 
+impl From<&RollupConfig> for HardForkConfiguration {
```

Comment:
these types are in the same crate.

this impl would be less horrible if we also add this natively to RollupConfig.

I guess `from<&>` wouldn't be an issue for `RollupConfig` because we assume this will likely be Arc'ed.
but still, very much dislike standalone from ref impls